internal/sub_reconcilers/reconcile_deployment.go

```diff
@@ -57,11 +59,33 @@ func (r *DeploymentReconciler) Reconcile(ctx context.Context, bestie *petsv1.Bes
 	log := r.Log.WithValues("deployment", logInfo)
 
 	// TODO wait for db to be seeded
+	job := &batchv1.Job{}
+	err := r.client.Get(ctx, types.NamespacedName{Name: bestie.Name + "-job", Namespace: bestie.Namespace}, job)
+	if err != nil && errors.IsNotFound(err) {
+		log.Info("No job initiated, job has not completed. Requeue to create deployment.")
+		delay := time.Second * time.Duration(5)
+		log.Info(fmt.Sprintf("will retry after waiting for %s", delay))
+		return ctrl.Result{RequeueAfter: delay}, nil
+	}
+
+	jobComplete := false
+	if job.Status.Succeeded >= *job.Spec.Completions {
+		jobComplete = true
+	}
+
+	if !jobComplete {
+		// If postgres is not ready yet, requeue after delay seconds.
+		delay := time.Second * time.Duration(15)
+		log.Info(fmt.Sprintf("job is instantiating, waiting for %s", delay))
+		// implement requeue after delay in subreconciler
+		return ctrl.Result{RequeueAfter: delay}, nil
```

Comment:
would remove the comments here and update the log to say something like database seeding job has not yet completed waiting for x seconds